tests/testthat/test-describe_distribution.R

```diff
@@ -349,6 +349,7 @@ test_that("multiple centralities work", {
 
 
 test_that("(multiple) centralities with CIs", {
+  skip_if_not_installed("bayestestR")
```

Comment:
I added this to the top of the file, because we literally used this in every test.

Reply:
agree, but we had this skip in 95% of all tests, so I thought it is probably ok to run this test only if bayestestR is available.